generatedocs/generated/documentation.go

```diff
@@ -216,6 +216,7 @@ var (
         "config.scheduler.settings.concurrentusers": { "Number of concurrent users to simulate. Allowed values are positive integers."  },  
         "config.scheduler.settings.executiontime": { "Test execution time (seconds). The sessions are disconnected when the specified time has elapsed. Allowed values are positive integers. `-1` means an infinite execution time."  },  
         "config.scheduler.settings.iterations": { "Number of iterations for each 'concurrent' user to repeat. Allowed values are positive integers. `-1` means an infinite number of iterations."  },  
+        "config.scheduler.settings.onlyinstanceseed": { "Disable session part of randomization seed. Defaults to `false` if omitted.","`true`: All users and sessions will have the same randomization sequence, only changing if `instance` flag is changed.","`false`: Normal randomization sequence, dependent on both `instance` parameter and current user session."  },  
```

Comment:
I guess this test is automatically generated and therefore does not need any review?

Reply:
both settingup.md and documentation.go are generated from params.json, so only need to comment on params.json